app/AppKernel.php

```diff
@@ -60,6 +54,21 @@ public function registerBundles()
         return $bundles;
     }
 
+    public function getRootDir()
+    {
+        return __DIR__;
+    }
+
+    public function getCacheDir()
+    {
+        return dirname(__DIR__).'/var/cache/'.$this->getEnvironment();
```

Comment:
If i remember correctly, `__DIR__ . '/../` is a bit faster than  `dirname(__DIR__)`, because it can be evaluated at compile time.